.github/workflows/release-tasks.yml

```diff
@@ -83,13 +83,23 @@ jobs:
       - name: Checkout LLVM
         uses: actions/checkout@v4
 
+      - name: Setup Cpp
+        uses: aminya/setup-cpp@v1
+        with:
+          compiler: llvm-16.0.6
+          cmake: true
+          ninja: true
+
       - name: Install dependencies
-        run: sudo apt-get install -y python3-setuptools
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-setuptools python3-psutil
 
       - name: Test lit
         run: |
-          cd llvm/utils/lit
-          python3 lit.py tests
+          mkdir build && cd build
+          cmake ../llvm -DCMAKE_BUILD_TYPE=Release -G Ninja
+          ninja -v -j $(nproc) check-lit
```

Comment:
To make debugging any failed tests easier, would you please consider adding the following to the environment when running check-lit?

```
export FILECHECK_OPTS='-dump-input-filter=all -vv -color'
```

Reply:
On second thought, I realize I don't know that I understand the context.  If this is run manually and can easily be re-run with different environment variables, then the person running it should decide whether to make this change.  Otherwise, defaulting to verbosity would be helpful for whoever has to understand test failures.

Reply:
It doesn't hurt to have extra verbosity.
Running this is not complex, but does require a specific setup.
I added included this suggestion in my last commit.

Reply:
Thanks!